src/App.jsx

```diff
@@ -26,6 +27,20 @@ const App = () => {
     setSearchTerm(cleanInput);
   };
 
+  // hardcoded values for transfer:
+  // list below shall be replaced from currency converter
+  const currencyBaseCode = "USD";
+  const currencyRecepientCode = "GBP";
+  const amountBase = 1000;
+  const amountReceived = 1359.5;
+  const fxRate = 1.3595;
```

Comment:
fxRate variable will clash with teh fxRate from teh currency converter, currency Converter has this : const [fxRate, setFxRate] = useState(currency[1].rate);